packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig

```diff
@@ -0,0 +1,20 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _removeText = props.removeText | default('Remove') -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'FileUploaderAttachment' -%}
+{%- set _removeClassName = _rootClassName ~ '__remove' -%}
```

Comment:
Busted :-)

Reply:
I'd like to have something shorter instead of `_spiritClassPrefix`, e.g. `cssPrefix`. Now I must admit it's easier to use it like this which I haven't been fan of… 🙈

Reply:
I am open to discussing this, but we should keep it the same everywhere. Now this component would be different from others. 

I suggest using the same approach here as we use elsewhere. And add this as a topic to our Thursday meetings and then if we decide to change how we do it, do It systematically everywhere.

Reply:
Now I see 🙈. I was looking on the `TextFieldBase` implementation during development, but there is a good reason for such construction. Fixed!